config.selenium-standalone-chrome.yaml

```
@@ -0,0 +1,21 @@
+web_environment:
```

Don't forget the `#ddev-generated` here
```suggestion
#ddev-generated

web_environment:
```